wavefront/server/packages/flo_cloud/flo_cloud/postgres/postgres.py

```
@@ -0,0 +1,303 @@
+            query, {'table_name': table_name, 'table_schema': self.schema}
+        )
+        return {'table_name': table_name, 'columns': columns}
+
+    def test_connection(self) -> bool:
+        try:
+            result = self.execute_query('SELECT 1')
+            success = len(result) > 0 and result[0][0] == 1
+            if success:
+                logger.info('Postgres connection test successful')
+            return success
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)